src/pages/index.mdx

```diff
@@ -1,16 +1,16 @@
 ---
 weight: 0
-title: "LiveChat Platform Docs"
-tagline: "The source of knowledge of the LiveChat Platform"
-desc: "LiveChat is an online customer service software with live support, help desk and web analytics capabilities. Browse the LiveChat Platform documentation to find out more."
+title: "Platform Docs"
+tagline: "The source of knowledge of the Platform Docs"
+desc: "Text Platform is an online customer service software with live support, help desk and web analytics capabilities. Browse the Platform Docs documentation to find out more."
```

Comment:
Now I think that changing it to `Platform Docs` would be a better idea🤔 I changed it as the first thing and later forgot about it as it's not visible on the preview.